.github/workflows/line-count.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           # Print total lines changed
           echo "Total lines changed (excluding Django migration files and tests directories): $total_lines_changed"
           
-          if [[ "$total_lines_changed" -gt 100 ]]; then
+          if [[ "$total_lines_changed" -gt 125 ]]; then
```

Comment:
This was done for the other open PR; this PR (excluding tests) is actually rather small